jwt/build.gradle

```diff
@@ -27,4 +27,24 @@ dependencies {
     testImplementation "com.github.uport-project.kotlin-common:test-helpers:$uport_kotlin_common_version"
     testImplementation "io.mockk:mockk:$mockk_version"
     testImplementation project(":jwt-test")
-}
\ No newline at end of file
+}
+
+
+apply plugin: 'jacoco'
+
+jacoco {
+    toolVersion = '0.7.5.201505241946'
+}
+
+jacocoTestReport {
+    reports {
+        xml.enabled = true
+        html.enabled = true
+        csv.enabled = true
+    }
+}
+
+test.finalizedBy jacocoTestReport
```

Comment:
This line does not need to exist since the task is called directly by the build script anyway.

Most of the time we don't actually want the report to be generated during testing, to be able to iterate faster.